tools/msvs/vswhere_usability_wrapper.cmd

```diff
@@ -28,5 +28,4 @@ for /f "usebackq tokens=*" %%i in (`vswhere %VSWHERE_ARGS%`) do (
 
 :no-vswhere
 endlocal
-echo could not find "vswhere"
```

Comment:
Can we keep this. Or upstream to https://github.com/node4good/msvs-com-helper

Reply:
>This removes that message, making output look nicer for VS2015 compilation.

Reply:
we just dropped "official" VS2015 support 🤷‍♂️ 

If you still want to keep output clean, I'd rather you change the call in `vcbuild` https://github.com/nodejs/node/blob/d217b2850efb9005819d55b697a37cbe5bd0003c/vcbuild.bat#L199 to:
```cmd
if defined DEBUG_HELPER (call tools\msvs\vswhere_usability_wrapper.cmd)
else (call tools\msvs\vswhere_usability_wrapper.cmd > nul)
```

Reply:
>If you still want to keep output clean, I'd rather you change the call in vcbuild 

I'd prefer to keep vcbuild clean too.

Reply:
Even without 2015, if no VS2017 is found, it will look like this:
```console
Looking for Visual Studio 2017
could not find "vswhere"
Failed to find a suitable Visual Studio installation.
Try to run in a "Developer Command Prompt" or consult
https://github.com/nodejs/node/blob/master/BUILDING.md#windows-1
```

I don't think it looks good, and the `could not find "vswhere"` is only helpful if you know vcbuild internals. We can use `DEBUG_HELPER` to debug vcbuild if needed.

Reply:
Ok, I'll move it upstream.